src/collections/_documentation/workflow/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Alerts
+sidebar_order: 1
+---
+
+Sentry users with admin permissions or higher can create one of two types of alerts: [Metric Alerts](#metric-alerts) or [Issue Alerts](#issue-alerts). You can use [Metric Alerts](#metric-alerts) for high-level monitoring of patterns, or fine-grained monitoring of individual events. An [Issue Alert](#issue-alerts) fires whenever any issue in the project matches specified criteria, such as a resolved issue re-appearing or an issue affecting many users. To confirm or set admin permissions, see **Project Settings > Alerts > New Alert**. 
+
+## Metric Alerts
+
+{% capture __alert_content -%}
+Metric Alerts are currently in beta. For more information about access to Metric Alerts, feel free to reach out at alerting-feedback@sentry.io.
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Metric alerts allow you to filter and set thresholds on all errors across a project. Use metric alerts for high-level monitoring of patterns or fine-grained monitoring of individual events. Metric alerts help express lightweight service-level objectives (SLOs) such as users affected by signup-page errors or volume of database errors.
```

Comment:
Long term I'd love if this linked to some guidance about best practices in SLOs.  For now is there a Glossary we can link to?
I think I'd add a comparison to Issue alerts (the thing people already know) possibly as another sentence: "To be notified about the impact of specific error, use issue alerts (link)"

Reply:
+1 to calling out issue alerts. I might change it to "impact of a specific _issue_"